### PR TITLE
add .net 5

### PR DESCRIPTION
**Pull request by ammarheidari**
add .net 5

### CODE REVIEW

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2;net4</TargetFrameworks>
+    <TargetFrameworks>netstandard2;net4;net5.0</TargetFrameworks>
```
Comment (falahati) [inline WindowsFirewallHelper/WindowsFirewallHelper.csproj:4]: isn't netstandard2 compatible with net5?

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj [hunk 2]
```diff
@@ -22,7 +22,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <Reference Condition="'$(TargetFramework)'=='net4'" Include="System.ServiceProcess"></Reference>
```
Comment (falahati) [inline WindowsFirewallHelper/WindowsFirewallHelper.csproj:24]: move this next to the new conditions too. being separated like this makes it hard to know what is exactly referenced and in what environment.

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj [hunk 3]
```diff
@@ -34,6 +33,16 @@
       <PackagePath>\</PackagePath>
     </Content>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2'">
```
Comment (falahati) [inline WindowsFirewallHelper/WindowsFirewallHelper.csproj:36]: these two conditions can be merged together.

**Comment by falahati** [comment]
Thanks for the PR.
I had some concerns that I have expressed in the previous comments as part of a review. please take a look and consider sharing your opinion on the issues.

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj (hunk 1 above)
Comment (ammarheidari) [reply WindowsFirewallHelper/WindowsFirewallHelper.csproj:4]: Assembly System.ServiceProcess.ServiceController, Version=4.2.3.0 not compatible with net5

Comment (falahati) [reply WindowsFirewallHelper/WindowsFirewallHelper.csproj:4]: hmm, and neither is v5 with netstandard2?

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj (hunk 3 above)
Comment (ammarheidari) [reply WindowsFirewallHelper/WindowsFirewallHelper.csproj:36]: Assembly System.ServiceProcess.ServiceController ver 4.7.0 not compatible with net5

File: WindowsFirewallHelper/WindowsFirewallHelper.csproj (hunk 1 above)
Comment (ammarheidari) [reply WindowsFirewallHelper/WindowsFirewallHelper.csproj:4]: only with netstandard2

**Pull request merged**